lib/image/image.go

```diff
@@ -22,6 +22,7 @@ import (
 	imgproxyerr "github.com/livesense-inc/fanlin/lib/error"
 	"github.com/livesense-inc/go-lcms/lcms"
 	"github.com/rwcarlsen/goexif/exif"
+	_ "github.com/strukturag/libheif-go"
```

Comment:
We can utilize the library just by importing it.

https://github.com/golang/go/blob/2869d55366a55d6a29a166523ac2383d0b2291df/src/image/format.go#L31-L42

https://github.com/strukturag/libheif-go/blob/55b3482bea15ac64bfa3039d543f913f9063936c/image_api.go#L82-L93